sysutils/borgbackup/Portfile

```
@@ -45,12 +47,23 @@ depends_lib-append  path:lib/libssl.dylib:openssl \
 depends_run-append  port:py${python.version}-msgpack \
                     port:py${python.version}-packaging
 
+# Run tests, but skip test_unix_socket, since it will fail because the path is
+# longer than the maximum length of a unix socket name when run in a MacPorts
+# build.
+depends_test-append port:py${python.version}-pytest-xdist \
+                    port:py${python.version}-pytest-benchmark
+
+test.run            yes
+test.args-append    -k 'not test_unix_socket' \
+                    -n ${build.jobs} \
+                    --pyargs borg.testsuite
+
 post-build {
     reinplace "s|sphinx-build|sphinx-build-${python.branch}|g" \
         ${worksrcpath}/docs/Makefile
     reinplace "s|guzzle_sphinx_theme|sphinx_rtd_theme|g" \
         ${worksrcpath}/docs/conf.py
-    reinplace "s|sphinx_rtd_theme.html_theme_path()|sphinx_rtd_theme.get_html_theme_path()|g" \
+    reinplace "s|sphinx_rtd_theme.html_theme_path()|\[sphinx_rtd_theme.get_html_theme_path()\]|g" \
```

Can you talk about why the square brackets are now needed?

If this `reinplace` is still needed, then the period in the regex should be escaped. Escaping is easier by using a `{}`-delimited string:

```suggestion
    reinplace {s|sphinx_rtd_theme\.html_theme_path()|[sphinx_rtd_theme.get_html_theme_path()]|g} \
```

---

> Can you talk about why the square brackets are now needed?

Is this to fix https://trac.macports.org/ticket/72684?

---

Yes. CI failed. I'm not exactly sure why, but the return value of `sphinx_rtd_theme.html_theme_path()` should apparently be a list but is a string. I didn't investigate why, I just fixed it.